extensions/ql-vscode/src/config.ts

```diff
@@ -635,3 +635,16 @@ export function getSkeletonWizardFolder(): string | undefined {
 export async function setSkeletonWizardFolder(folder: string | undefined) {
   await SKELETON_WIZARD_FOLDER.updateValue(folder, ConfigurationTarget.Global);
 }
+
+/**
+ * Option to turn on/off ability to autogenerate QL packs. Values are "Yes" / "No" / "Never ask again"
```

Comment:
Unfortunately this comment of the expected values doesn't match the definition in [`package.json`](https://github.com/github/vscode-codeql/pull/2359/files#diff-be4c5a31505df7de81fa4900e84e66c04d1a6bca9b3da50f8061d2cb2bfbbde1R353)

Could we define a typescript type for it so we aren't dealing in raw strings? Either an enum or a string union would be great.